### ISSUE TITLE
BUG: Pagination accessability

### ISSUE THREAD

**Issue by maqsudtolipov** [adadychanka/CFS#62]
Table pagination is missing accessibility wrapper components. I forgot to add.

Resolved directly into inside pre-stating
